cli/core/handlers/excel_mixins/read.py

```
@@ -0,0 +1,105 @@
+from typing import Any
+
+from cli.core.handlers.excel_mixins.types import (
+    ColumnPatterns,
+    SheetData,
+    SheetDataGenerator,
+)
+from openpyxl.utils import get_column_letter
+from openpyxl.worksheet.worksheet import Worksheet
+
+
+class ExcelReadMixin:  # noqa: WPS214
```

Ruff (0.15.12)

[warning] Invalid rule code in `# noqa`: WPS214

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)